src/main/java/com/github/manolo8/darkbot/gui/tree/editors/ImagePicker.java

```diff
@@ -0,0 +1,76 @@
+package com.github.manolo8.darkbot.gui.tree.editors;
+
+import com.github.manolo8.darkbot.config.ImageWrapper;
+import com.github.manolo8.darkbot.gui.AdvancedConfig;
+import eu.darkbot.api.config.util.OptionEditor;
+import com.github.manolo8.darkbot.gui.utils.Strings;
+import eu.darkbot.api.config.ConfigSetting;
+
+import javax.swing.*;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import java.awt.*;
+import java.io.File;
+
+public class ImagePicker extends JButton implements OptionEditor<ImageWrapper> {
+    private JFileChooser fc;
+    private ImageWrapper image;
+
+    public ImagePicker(){
+        putClientProperty("JButton.buttonType", "square");
```

Comment:
i could be wrong but i *think* this may no longer be needed, we enforce square-ness on a higher level already, try to remove it